algebird-core/src/main/scala/com/twitter/algebird/Ring.scala

```diff
@@ -202,8 +243,14 @@ object Ring extends GeneratedRingImplicits with ProductRings with RingImplicits0
   implicit def doubleRing: Ring[Double] = DoubleRing
   implicit def jdoubleRing: Ring[JDouble] = JDoubleRing
   implicit def indexedSeqRing[T: Ring]: Ring[IndexedSeq[T]] = new IndexedSeqRing[T]
+  /**
+   * This is actually a Rng but we leave the unsafe Ring for legacy reasons
+   */
   implicit def mapRing[K, V](implicit ring: Ring[V]): Ring[Map[K, V]] =
```

Comment:
sadly no, because algebra does not use the import-to-get-instances style. If we did that it would break source compatibility for people that expect to get Map rings.